exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/xsort/TestSimpleExternalSort.java

```diff
@@ -148,10 +149,10 @@ public void sortOneKeyDescendingExternalSortLegacy() throws Throwable {
 
   private void sortOneKeyDescendingExternalSort(boolean testLegacy) throws Throwable {
     FixtureBuilder builder = ClusterFixture.builder()
-        .configProperty(ExecConstants.EXTERNAL_SORT_SPILL_THRESHOLD, 4)
-        .configProperty(ExecConstants.EXTERNAL_SORT_SPILL_GROUP_SIZE, 4)
-        .configProperty(ExecConstants.EXTERNAL_SORT_BATCH_LIMIT, 4)
-        .configProperty(ExecConstants.EXTERNAL_SORT_DISABLE_MANAGED, false)
+        .configProperty(OptionValidator.OPTION_DEFAULTS_ROOT + ExecConstants.EXTERNAL_SORT_SPILL_THRESHOLD, 4)
```

Comment:
These options are boot options, not runtime options, so no need for the prefix (unless someone changed something.)

Reply:
Done, thanks for catching.